validate_config.sh

```diff
@@ -10,7 +10,7 @@ function validate_config_yml {
     tmp_json=${config}.json
     yq $config -o json > $tmp_json
     set +e
-    jsonschema --instance $tmp_json $SCHEMA
+    check-jsonschema --schemafile $SCHEMA $tmp_json
```

Comment:
Can you please check if that tool exists and if not raise an error message to rerun the ./toolset/script/install.sh ?